packages/docs/pages/margin.tsx

```diff
@@ -1,55 +1,67 @@
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Responsive breakpoints">
-            <Text>
-              You can also specify margins with an object to handle different margins with different{' '}
-              <NextLink href="/breakpoints">breakpoints</NextLink>. All values must be of{' '}
-              <NextLink href="/spacing">Spacing</NextLink> type.
-            </Text>
-
-            <CodePreview>
-              {/* jsx-to-string:start */}
-              <Button margin={{ mobile: 'none', tablet: 'medium', desktop: 'xLarge' }}>Example</Button>
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-        </>
-      ),
-    },
-    {
-      id: 'props',
-      title: 'Props',
-      render: () => <MarginPropTable />,
-    },
-  ];
-
   return (
     <>
       <H1>Margin</H1>
 
-      <PageNavigation items={items} />
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          BigDesign provides the <Code primary>margin</Code> prop which can be used in some of our components.
+        </Text>
+      </Panel>
+
+      <Panel header="Implementation" headerId="implementation">
+        <ContentRoutingTabs
+          id="implementation"
+          routes={[
+            {
+              id: 'basic',
+              title: 'Basic',
+              render: () => (
+                <Fragment key="basic">
+                  <Text>
+                    The following example showcases the simplest way to use the <Code primary>margin</Code> prop by
+                    giving it a <NextLink href="/spacing">Spacing</NextLink> value.
+                  </Text>
+                  <CodePreview>
+                    {/* jsx-to-string:start */}
+                    <Button margin="medium">Example</Button>
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </Fragment>
+              ),
+            },
+            {
+              id: 'responsive-breakpoints',
+              title: 'Responsive breakpoints',
+              render: () => (
+                <Fragment>
+                  <Text>
+                    You can also specify margins with an object to handle different margins with different{' '}
+                    <NextLink href="/breakpoints">breakpoints</NextLink>. All values must be of{' '}
+                    <NextLink href="/spacing">Spacing</NextLink> type.
+                  </Text>
+
+                  <CodePreview>
+                    {/* jsx-to-string:start */}
+                    <Button margin={{ mobile: 'none', tablet: 'medium', desktop: 'xLarge' }}>Example</Button>
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </Fragment>
+              ),
+            },
+          ]}
+        />
+      </Panel>
+
+      <Panel header="Props" headerId="props">
+        <MarginPropTable renderPanel={false} />
+      </Panel>
     </>
```

Comment:
Note: 
I was thinking of adding the following in the `Guideline` section: 
All values must be of{' '}<NextLink href="/spacing">Spacing</NextLink> type.

However, I feel that the `<MarginPropTable/>` shows the Spacing as the type, so might not be necessary to mention it in the `Guideline` section. 

Other than that, I think there is nothing important that I can think of 🤔 ... Does someone has a suggestion of what could be added in this section?

Reply:
I think it's safe to omit that sentence. `Spacing` is perfect enough and I have some future plans to show what values you can use.